### ISSUE TITLE
Debian build instructions are inaccurate

### ISSUE THREAD

**Issue by marmistrz** [openucx/ucx#4094]
README tells to use `dpkg-buildpackage` but this won't work, because there exists no debian/control & debian/changelog. debian/control.in & debian/changelog.in exist instead.

How should one actually build the Debian package?

**Comment by shamisp** [comment]
I don't think we actually test Debian packages. @yosefe may know.

**Comment by mike-dubman** [comment]
We test with debian.
Files you mention are generated by configure process.
Once autogen and configure are done - you can use dpkg

**Comment by shamisp** [comment]
@marmistrz can you please confirm. Thanks @mike-dubman .

**Comment by marmistrz** [comment]
Thanks, I will, probably on Monday.

**Comment by marmistrz** [comment]
>  Once autogen and configure are done - you can use dpkg
This is right. Thanks. But it appears that even with `DEB_BUILD_OPTIONS='parallel=4'` only one gcc process is spawned. (i.e. make is not parallel)
Probably using `dh --parallel` or bumping `debian/compat` to `10` should do the trick, according to this [AskUbuntu post](https://askubuntu.com/questions/337093/how-to-run-parallel-make-with-debuild)

**Comment by mike-dubman** [comment]
@yosefe
